src/lib/css-parse.html

```diff
@@ -162,6 +163,7 @@
       mixinApply: /@apply[\s]*\([^)]*?\)[\s]*(?:[;\n]|$)?/gim,
       varApply: /[^;:]*?:[^;]*var[^;]*(?:[;\n]|$)?/gim,
       keyframesRule: /^@[^\s]*keyframes/,
+      multipleSpaces: /\s+/g

```

Comment:
A bit nitpicky, but as pointed out in the other PR, `\s` matches whitespace. In my eyes it would be more logical to name this regex `whitespace`.

Same for the id name and description in the testcase.